vep/test/test-dataproc-wrapper.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env python3
+
+import click
+import hail as hl
+from cpg_workflows.batch import get_batch
+
+
+@click.command()
+@click.argument('vep_version')
+def main(vep_version: str):
+    """
+    Submit `test-dataproc-script.py` to cluster.
```

Comment:
should this script be submitted using analysis runner? If so I can look at adding a couple of collect/assert statements so that the job has in-built checking (e.g. failed status when the annotations were present, but incorrect)

Reply:
Yeah, should be run with the analysis-runner